frame/support/src/traits.rs

```diff
@@ -83,4 +83,4 @@ mod voting;
 pub use voting::{CurrencyToVote, SaturatingCurrencyToVote, U128CurrencyToVote};
 
 // for backwards-compatibility with existing imports
-pub use max_encoded_len::MaxEncodedLen;
+pub use codec::MaxEncodedLen;
```

Comment:
ideally we should also remove this reexport before publishing

Reply:
I did https://github.com/paritytech/substrate/pull/9186 we could merge after this PR maybe.

Reply:
Perhaps merge #9186 into this PR?

Reply:
Done now